deploy/crds/ohmyglb.absa.oss_v1beta1_gslb_cr.yaml

```diff
@@ -29,3 +29,5 @@ spec:
             path: /
   strategy:
     type: roundRobin # Use a round robin load balancing strategy, when deciding which downstream clusters to route clients too
+    splitBrainThresholdSeconds: 300 # Threshold after which external cluster is filtered out from delegated zone when it doesn't look alive
+    dnsTtlSeconds: 30 # TTL of DNS A records
```

Comment:
it also affects split brain TXT record
```
_, err := objMgr.CreateTXTRecord(heartbeatTXTName, edgeTimestamp, gslb.Spec.Strategy.DNSTtlSeconds, "default")
```
So maybe it is worth to rephrase `TTL for DNS A and TXT records` .
Or more generic `TTL value for automatically created DNS records` as we might add more types in future